### PR TITLE
add resources for users, profiles, operator accounts and sms account

### PR DESCRIPTION
**Pull request by mbarrailh**

### CODE REVIEW

File: .jshintrc [hunk 1]
```diff
@@ -1,13 +1,11 @@
...
-    // Example: https://github.com/jshint/jshint/blob/master/examples/.jshintrc
-    // Documentation: http://www.jshint.com/docs/
     "undef": true,
     "unused": "vars",
     "laxbreak": true,
     "node": true,
     "strict": false,
-    "sub": true,
+    "sub": true,,
+    esversion": 6
```
Comment (sebz) [inline .jshintrc]: I have another branch for the switch to ES6, can you remove this property?

File: lib/airvantage.js [hunk 2]
```diff
@@ -1,3 +1,4 @@
+"use strict";
```
Comment (sebz) [inline lib/airvantage.js]: Should not be required in ES5.

**Review by sebz — CHANGES_REQUESTED** [review]

File: .jshintrc [hunk 3]
```diff
@@ -1,7 +1,4 @@
 {
-    // Details: https://github.com/victorporof/Sublime-JSHint#using-your-own-jshintrc-options
-    // Example: https://github.com/jshint/jshint/blob/master/examples/.jshintrc
-    // Documentation: http://www.jshint.com/docs/
```
Comment (jlescot) [inline .jshintrc]: I guess we might keep these comments...

File: lib/airvantage.js [hunk 4]
```diff
@@ -634,4 +634,4 @@ function getArgumentsAsString(args) {
         }
     });
     return response.join(" ");
-}
+}
```
Comment (jlescot) [inline lib/airvantage.js]: ⚠️ Code formatter used to add a blank line at the end of the file...

**Pull request merged**